content/_data/changelogs/lts.yml

```
@@ -9642,6 +9642,9 @@
 
 - version: "2.414.3"
   date: 2023-10-18
+  banner: >
+      Short tags (without "jdk" in them) like for example <code>jenkins/jenkins:2.414.3-alpine</code> are using Java 17 and not Java 11 like previously.
+      If you need to keep using Java 11 use tags like <code>jenkins/jenkins:2.414.3-jdk11</code>.
```

```suggestion
      Short tags (without "jdk" in them) such as <code>jenkins/jenkins:2.414.3-alpine</code> are using Java 17 and not Java 11 like previously.
      If you need to keep using Java 11, use tags like <code>jenkins/jenkins:2.414.3-jdk11</code>.
```
Suggestion for a bit of grammar update/syntax matching.